Dockerfile

```diff
@@ -9,12 +9,12 @@ RUN npm run build
 
 WORKDIR /usr/app/dist/src/examples/poi-crosschecker
 
-ENV GRAPH_NODE_HOST "host.docker.internal"
-ENV ETH_NODE "host.docker.internal:8545"
+ENV REGISTRY_SUBGRAPH "https://api.thegraph.com/subgraphs/name/juanmardefago/gossip-network-subgraph"
 ENV NETWORK_URL "https://gateway.testnet.thegraph.com/network"
 ENV TERM "xterm-256color"
-ENV REGISTRY_SUBGRAPH "https://api.thegraph.com/subgraphs/name/juanmardefago/gossip-network-subgraph"
-ENV INDEXER_MANAGERMENT_SERVER "host.docker.internal:18000"
+ENV GRAPH_NODE host.docker.internal:8030
+ENV ETH_NODE host.docker.internal:8545
+ENV INDEXER_MANAGEMENT_SERVER_PORT host.docker.internal:18000
```

Comment:
Maybe we can remove these from the Dockerfile now? Given that we're passing a `.env` file in docker-compose and also to the unit tests, it seems like duplication

Given the current setup and with deleted env vars from the Dockerfile we can still run just one "good indexer" instance by running `docker-compose up good-indexer` and we can run the integration test scenario with just `docker-compose up` (we should define npm commands for both these cases too, but that can be for another PR as it's just a QOL improvement) 😁

Reply:
They are default values provided to the image, and they are replaced by specified duplicated env var from docker compose, so I would prefer to keep them as a baseline?

Reply:
that makes sense, it's a Radio implementation detail anyway so we shouldn't overthink it